### ISSUE TITLE
Binary artifact in flutter/packages repository

### ISSUE THREAD

**Issue by godofredoc** [flutter/flutter#123835]
`packages/rfw/example/wasm/logic/calculator.wasm:1` was checked into the packages repository.

# Remediation:

* Remove the generated executable artifacts from the repository.
* Build from source.

# Severity: High

Details:

# Risk: High (non-reviewable code)

**Comment by godofredoc** [comment]
Binary was included in this PR: https://github.com/flutter/packages/pull/452

\cc @Hixie

**Comment by Hixie** [comment]
Yeah, it's part of the demo for RFW. It needs to be downloadable for the demo to work. If there's a way we can compile the artifact in CI to verify that it's the right bytes, that would make sense...

**Comment by Hixie** [comment]
I'm going to leave this as-is for now, maybe we can fix it when we fix #129385, that way we don't have to worry about introducing a new build pipeline into the CI for this repo just for a tiny example.
